kanvas/src/commonMain/kotlin/xml/Scientific.kt

```diff
@@ -0,0 +1,88 @@
+package com.juul.krayon.kanvas.xml
+
+import kotlin.math.log
+import kotlin.math.nextUp
+import kotlin.math.pow
+import kotlin.math.roundToInt
+
+/**
+ * Returns a string representing this number in scientific e notation.
```

Comment:
Annoying that this was required for the JS target

Reply:
I totally understand having the default `toString` match the platform, but it's quite sad Kotlin has no built in way of formatting numbers consistently.